index.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 const fp = require('fastify-plugin')
+const assert = require('assert')
 
 function underPressure (fastify, opts, next) {
```

Comment:
it's better if you rewrite this using an async function and remove next.. using an `await` for the health check, so you can avoid that bad comment in the end.